src/main/webapp/styles/tutorialstyle.css

```diff
@@ -11,43 +11,46 @@
 }
 
 .tutorials-vid {
-  width: 448px;
-  height: 252px;
-  border: 0;
+  width: 87%;
+  height: 45%;
```

Comment:
There's quite a mix of css sizing based on percentages, em's, and pixels!

I don't have a specific request here, but might be worth looking into the differences and thinking about which one is best suited to your app. It sounds like you guys are interested in making the page responsive to different form factors (mobile vs desktop, etc) so using %s may be the right move. Here's a decent primer on the differences if you want to take a look: https://kyleschaeffer.com/css-font-size-em-vs-px-vs-pt-vs-percent

Another thought on CSS - I see a few places where we're reusing CSS rules (like colors, etc). Like the HTML files I mentioned above, it can be really hard to keep CSS rules in sync if they're duplicated in a bunch of places. A tool like Sass (https://sass-lang.com/) could really help with that. It lets you define variables/mixins/etc which you can import into multiple CSS rules. Sass is really powerful and used all over the place, so might be worth checking out if you haven't seen it before.